src/Navigation/Drawer.tsx

```diff
@@ -115,42 +118,67 @@ const Drawer = ({ username }: { username: string }) => {
-            <CIcon icon={cibGithub} />
-          </div>
           <ul>
             <li>
               <details open>
-                <summary className="font-bold text-slate-800">
+                <summary className="font-bold text-slate-800 text-xl">
                   Bookmarked
                 </summary>
-                <ul>
-                  <li>
-                    <Link to="/">julianna-venter</Link>
+                <ul className="text-lg">
+                  <li className="flex flex-row gap-2 cursor-pointer">
+                    <Link
+                      to="/"
+                      className="p-0 pl-4 flex justify-center items-center"
+                    >
+                      julianna-venter
+                    </Link>
+                    <div
+                      id="bookmarkIcon"
+                      tabIndex={0}
+                      role="button"
+                      className="btn btn-ghost btn-circle"
+                    >
+                      <svg
+                        xmlns="http://www.w3.org/2000/svg"
+                        viewBox="0 0 24 24"
+                        fill="currentColor"
+                        className="w-6 h-6"
+                      >
+                        <path
+                          fillRule="evenodd"
+                          d="M6.32 2.577a49.255 49.255 0 0 1 11.36 0c1.497.174 2.57 1.46 2.57 2.93V21a.75.75 0 0 1-1.085.67L12 18.089l-7.165 3.583A.75.75 0 0 1 3.75 21V5.507c0-1.47 1.073-2.756 2.57-2.93Z"
+                          clipRule="evenodd"
+                        />
+                      </svg>{" "}
+                    </div>
                   </li>
                 </ul>
               </details>
             </li>
             <li>
               <details open>
-                <summary className="font-bold text-slate-800">Users</summary>
-                <ul>
-                  <li>
+                <summary className="font-bold text-slate-800 text-xl">
+                  Users
+                </summary>
+                <ul className="text-lg">
+                  <li className="cursor-pointer">
                     <Link to="/">yanevdw</Link>
                   </li>
-                  <li>
+                  <li className="cursor-pointer">
                     <Link to="/">meggie-h</Link>
                   </li>
                 </ul>
               </details>
             </li>
           </ul>
-          <Link
-            to="/"
-            className="w-16 h-16 bg-dark-off-white rounded-full p-3 absolute bottom-4 right-4 hover:bg-primary-blue"
-          >
-            <CIcon icon={cilSearch} className="hover:text-white" />
-          </Link>
+          {username && (
+            <Link
+              to="/"
+              className="w-16 h-16 bg-primary-blue/20 text-primary-blue rounded-full p-4 absolute bottom-4 right-4 hover:bg-secondary-orange hover:text-white cursor-pointer"
```

Comment:
`Line 177`: I am unsure if this exists `bg-primary-blue/20`, perhaps a typo? Otherwise, disregard this comment.

Reply:
yeah, /20 indicates opacity